_posts/2021-04-18-api-conversion.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Path to CRD v1"
+date: 2021-04-18
+author: jakub_stejskal
+---
+In Strimzi 0.22 we introduced new API version `v1beta2` which will replace `v1beta1` / `v1alpha1` APIs and make Strimzi ready for Kubernetes 1.22.
+This blog post will show you how to perform the conversion and get ready for next Strimzi releases.
+
+<!--more-->
+
+Even with these changes, there are still some disadvantages:
+* For upgrade, roll out the new brokers while first using the older `log.message.format.version` or `inter.broker.protocol.version` and only afterwards change to the new versions for message format and inter-broker protocol.
+* Downgrade won't be executed when new `log.message.format.version` or `inter.broker.protocol.version` are already used.
+  You need to change configuration to use same `log.message.format.version` and `inter.broker.protocol.version` as older Kafka.
+
+You can find further information in our [documentation](https://strimzi.io/docs/operators/0.22.1/full/deploying.html#proc-upgrade-cli-tool-crds-str).
```

Comment:
This should not be here, or? Apart from the link which should be maybe at the end?

Reply:
ye, it's leftover